services/TipiVariable.py

```diff
@@ -227,7 +244,7 @@ def store(self, caller_guid):
         f = open(runtime_dir + str(caller_guid), "w")
         
         for key, val in self.ti_vars.items():
-            if len(key):
+            if len(key) and len(val):
```

Comment:
So with this, there is no way for a key's value to be removed...    how do programmers using this system decide they no longer need variable 'whatsit' and remove it from their storage?  

You should eventually add a 'D' for delete key option.

Reply:
I thought about this, and the way to delete a var is to store it with an empty value. Since the “registry” file gets written fresh each time, an empty var would not be saved. I did a little testing to make sure I didn’t break the system, but I’ll do a test to make sure that deleting is possible. You’re right, an explicit delete is a good idea to make it clear.